frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -21,12 +28,26 @@ import {
   dataVolumeManifest,
 } from './utils/mocks';
 
+function resolveStorageDataAttribute(configMap: any, attribute: string): string {
+  if (Object.prototype.hasOwnProperty.call(configMap, `${STORAGE_CLASS}.${attribute}`)) {
+    return configMap[`${STORAGE_CLASS}.${attribute}`];
+  }
+  return configMap[attribute];
+}
+
 describe('Kubevirt create VM using wizard', () => {
   const leakedResources = new Set<string>();
+  const kubevirtStorage = getResourceObject(
+    KUBEVIRT_STORAGE_CLASS_DEFAULTS,
+    KUBEVIRT_PROJECT_NAME,
+    'configMap',
+  );
   const testDataVolume = dataVolumeManifest({
     name: `toclone-${testName}`,
     namespace: testName,
     sourceURL: basicVmConfig.sourceURL,
+    accessMode: resolveStorageDataAttribute(kubevirtStorage.data, 'accessMode'),
```

Comment:
I think this would look prettier

```suggestion
    accessMode: resolveStorageDataAttribute(kubevirtStorage, 'accessMode'),
```

Reply:
Done